go.mod

```diff
@@ -1,21 +1,18 @@
-module github.com/anz-bank/sysl-go-comms
+module github.com/anz-bank/sysl-go
 
 require (
+	github.com/anz-bank/sysl-go-comms v0.0.0-20200313054710-9d2033a94714 // indirect
```

Comment:
This looks like a problem.

Reply:
Addressed in https://github.com/anz-bank/sysl-go/pull/14/commits/70cf7698bd11574944d9899c603cf44353ab29c7

Reply:
Resolved!